packages/components/src/molecules/Panels/ResizableColumnsPanel.tsx

```diff
@@ -25,9 +18,22 @@ const ResizableColumnsPanel: React.FC<ResizableColumnsPanelType> = (props) => {
   return (
     <ReflexContainer orientation="vertical" windowResizeAware style={{ height, width }}>
       {left && (
-        <ReflexElement minSize={minPaneWidth} onStopResize={onStopResizeLeft} flex={layout?.left}>
-          <S.LeftPane>{left}</S.LeftPane>
-        </ReflexElement>
+        <StyledLeftReflexElement
+          $leftClosable={leftClosable}
+          minSize={minPaneWidth}
+          onStopResize={onStopResizeLeft}
+          flex={layout?.left}
+        >
+          <StyledLeftPane $leftClosable={leftClosable}>
+            {left}
+            {leftClosable && (
+              <PaneCloseIcon
+                onClick={onCloseLeftPane}
+                containerStyle={{ position: "absolute", top: 20, right: -10, zIndex: 200 }}
```

Comment:
better to get zIndex from a layout otherwise we can confuse index levels in future like layout.zIndex.1, layout.zIndex.2, layout.zIndex.max